.github/workflows/system-tests.yml

```diff
@@ -8,35 +8,71 @@ on:
   schedule:
     - cron:  '00 04 * * 2-6'
 
+env:
+  REGISTRY: ghcr.io
+
 jobs:
   system-tests:
+    strategy:
+      matrix:
+        include:
+          - library: ruby
+            weblog-variant: rack
+          - library: ruby
+            weblog-variant: sinatra14
+          - library: ruby
+            weblog-variant: sinatra20
+          - library: ruby
+            weblog-variant: sinatra21
+          - library: ruby
+            weblog-variant: rails32
+          - library: ruby
+            weblog-variant: rails40
+          - library: ruby
+            weblog-variant: rails41
+          - library: ruby
+            weblog-variant: rails42
+          - library: ruby
+            weblog-variant: rails50
+          - library: ruby
+            weblog-variant: rails51
+          - library: ruby
+            weblog-variant: rails52
+          - library: ruby
+            weblog-variant: rails60
+          - library: ruby
+            weblog-variant: rails61
+          - library: ruby
+            weblog-variant: rails70
     runs-on: ubuntu-latest
+    name: System Tests (${{ matrix.weblog-variant }})
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           repository: 'DataDog/system-tests'
+          ref: 'use-system-test-apps-for-ruby'
```

Comment:
This should go away as soon as https://github.com/DataDog/system-tests/pull/163 is merged.